antora-ui-camel/src/css/doc.css

```diff
@@ -634,3 +636,22 @@ kbd,
 .stretch {
   width: 100%;
 }
+
+/* for mobile screens */
+@media screen and (max-width: 626px) {
+  #preamble + .sect1,
+  .doc .sect1 + .sect1 {
+    overflow-wrap: break-word;
+  }
+}
+
+@media screen and (max-width: 1023px) and (min-width: 480px) {
+  .doc > h1.page:first-child,
+  .static > h1:first-child {
+    font-size: inherit;
+  }
+
+  .doc h2:not(.discrete) {
+    overflow-wrap: break-word;
+  }
+}
```

Comment:
I think this needs to be rethought: these are some very targeted rules, I'd prefer if we came up with a principle rather than a fix for specific instances of text. This will be very difficult to maintain with the text changing.

Reply:
In this, we have a different font size so I use this media query for some targeted areas font size.
I made changes in this and sent it again.